exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js

```diff
@@ -24,7 +24,7 @@ module.exports = function() {
 
   this.Given(/^an instance of this service$/, function(done) {
     this.process = new ExoService({ serviceName: '_____serviceName_____',
-                                    exocomHost: 'localhost'
+                                    exocomHost: 'localhost',
```

Comment:
commas are optional in LiveScript, you can remove this one and the one on the line above